frontend/chat-plugin/src/airyRenderProps/AiryInputBar/index.tsx

```diff
@@ -13,7 +13,7 @@ const AiryInputBar = (props: Props) => {
     setMessageString('');
   }, []);
 
-  const resizeTextarea = e => {
+  const resizeTextarea = (e: any) => {
```

Comment:
Is `any` not the default? 🤔

Reply:
Also since this is a InputChange, we could add the correct type, or do I miss something?

Reply:
Actually I was trying to type it right, but the event types in preact are a bit weird and it wasn't working... I will try again @bitboxer @chrismatix